testing/peervpn/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Contributor: Markus Juenemann <markus@juenemann.net>
+# Maintainer: Markus Juenemann <markus@juenemann.net>
+pkgname=peervpn
+pkgver=0.044
+pkgrel=0
+pkgdesc="PeerVPN builds virtual ethernet networks between multiple computers"
+url="https://peervpn.net"
+arch="all"
+license="GPL3"
+depends="openssl zlib"
+makedepends="$depends openssl-dev zlib-dev linux-headers"
+install=""
+subpackages=""
+source="
+        $url/files/$pkgname-0-044.tar.gz
```

Comment:
We normally don't use $url here, because $url goes to package database. Maybe just expand it here?
You could also use ${pkgver/./-} or similar for the version.